src/Patterns/GetInformed.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * Get Informed pattern class.
+ *
+ * @package P4\MasterTheme\Patterns
+ * @since 0.1
+ */
+
+namespace P4\MasterTheme\Patterns;
+
+/**
+ * Class Get Informed.
+ *
+ * @package P4\MasterTheme\Patterns
+ */
+class GetInformed extends BlockPattern
+{
+    /**
+     * @inheritDoc
+     */
+    public static function get_name(): string
+    {
+        return 'p4/get-informed-pattern-layout';
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public static function get_config($params = []): array
+    {
+        return [
+            'title' => 'Get Informed',
+            'blockTypes' => [ 'core/post-content' ],
+            'categories' => [ 'layouts' ],
+            'content' => '
+				<!-- wp:planet4-block-templates/get-informed ' . wp_json_encode($params, \JSON_FORCE_OBJECT) . ' /-->
```

Comment:
Should we use `esc_html` to ensure the JSON string is properly escaped for safe HTML output?

```suggestion
				<!-- wp:planet4-block-templates/get-informed ' . esc_html(\wp_json_encode($params, \JSON_FORCE_OBJECT)) . ' /-->
```

Reply:
I'm not sure if this is necessary since in this case you're encoding an object which passed as props. We don't need to pass HTML code within props.